src/global_distrib/mod_global_distrib_mapping.erl

```diff
@@ -163,8 +163,8 @@ deps(Host, Opts) ->
 %% Hooks implementation
 %%--------------------------------------------------------------------
 
--spec session_opened(mongoose_acc:t(), ejabberd_sm:sid(), UserJID :: jid:jid(), Info :: list()) ->
-    mongoose_acc:t().
+-spec session_opened(Acc, ejabberd_sm:sid(), UserJID :: jid:jid(), Info :: list()) ->
+    Acc when Acc :: any().
```

Comment:
why `any()` type?

Reply:
The same reason as for `ejabberd_auth_annonymous`